src/status_im/ui/screens/chat/views.cljs

```diff
@@ -70,36 +70,45 @@
   [{{:keys [value]} :row}]
   [message-datemark/chat-datemark-mobile value])
 
-(defview gap [{:keys [ids]}]
+(defview gap
+  [ids idx list-ref in-progress? connected?]
+  [react/view {:align-self          :stretch
```

Comment:
move to styles?

Reply:
ehm, let me finish with desktop first, and other related PRs. I'm sure it will be changed 2-3 times during next PRs. It's just simplier to keep everything in one place for now